src/main/java/org/broadinstitute/hellbender/tools/AnnotatePairOrientation.java

```diff
@@ -47,7 +48,7 @@
             doc = "Output Somatic SNP/Indel VCF file with additional annotations.",
             shortName = StandardArgumentDefinitions.OUTPUT_SHORT_NAME,
             fullName = StandardArgumentDefinitions.OUTPUT_LONG_NAME)
-    protected File outputFile;
+    protected String outputVCF;
```

Comment:
Next time it would be useful if you made these suggestions on the [initial single-tool change](https://github.com/broadinstitute/gatk/pull/5378) that sets the pattern rather than after I have applied the pattern to every single remaining tool. This would ensure I don't need to go back and redo the sweeping change.

Reply:
Point taken, though this class didn't exist until a few weeks ago, well after we had been through several few rounds of review on the initial PR.